src/main.js

```diff
@@ -117,9 +121,9 @@ $(document).ready(() => {
       }
     }
 
-    function tryLoadRepo(reload) {
+    function tryLoadRepo(reload, isChangedBranch = false, branch) {
       const token = octotree.getAccessToken();
-      adapter.getRepoFromPath(currRepo, token, (err, repo) => {
+      adapter.getRepoFromPath(currRepo, token, async (err, repo) => {
```

Comment:
We don't need to use `async` if there is no `await` in the body

Reply:
hmm, thanks didn't notice it :).